spec/_attachments/ic.did

```diff
@@ -184,6 +192,14 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  metrics : (record {
+    subnet_id : principal;
```

Comment:
```suggestion
    subnet_id : principal;
    minimal_timestamp_nanos: nat64;
```
see discussion on design slides https://docs.google.com/presentation/d/1KXIZtYtn658L2mDKrgmL502jeBo1QbHoJRYYAg7Pu68/edit?disco=AAAA4HT7no4

To ensure the size of the response is small enough to justify charging the method invocation cost only we want to return only the records with the smallest timestamp_nanos > minimal_timestamp_nanos

Reply:
I like your suggestions!